test/check-networking

```diff
@@ -45,6 +45,7 @@ class TestNetworking(MachineCase):
             element = self.browser.text("#networking-interfaces tr:contains('%s')" % iface)
             for state in valid_states:
                 if state in element:
+                    self.browser.wait_visible("#networking-interfaces tr:contains('%s')" % iface)
                     return
             time.sleep(0.5)
```

Comment:
So what I wanted to say with this is that a loop like

```
while not "foo" in b.text("#bla"')
  sleep(1)
```

will not make progress in the browser, and should always return the same value from b.text(), forever.